dbt/adapters/firebolt/connections.py

```diff
@@ -181,9 +185,7 @@ def commit(self) -> None:
 
     def cancel(self, connection: Connection) -> None:
         """Cancel the last query on the given connection."""
-        raise dbt.exceptions.NotImplementedError(
-            '`cancel` is not implemented for this adapter!'
-        )
+        raise NotImplementedError('`cancel` is not implemented for this adapter!')
```

Comment:
Is this one about reverting the last operation or about canceling the ongoing query? Can we implement this if latter?